src/glue/auth_checker.hpp

```diff
@@ -39,28 +39,28 @@ class AuthChecker : public query::AuthChecker {
 
 class FineGrainedAuthChecker : public query::FineGrainedAuthChecker {
  public:
-  explicit FineGrainedAuthChecker(auth::User user);
+  explicit FineGrainedAuthChecker(auth::User user, const memgraph::query::DbAccessor &dba);
 
-  bool Accept(const memgraph::query::DbAccessor &dba, const query::VertexAccessor &vertex, memgraph::storage::View view,
-              query::AuthQuery::FineGrainedPrivilege fine_grained_permission) const override;
+  bool Has(const query::VertexAccessor &vertex, memgraph::storage::View view,
+           query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
-  bool Accept(const memgraph::query::DbAccessor &dba, const query::EdgeAccessor &edge,
-              query::AuthQuery::FineGrainedPrivilege fine_grained_permission) const override;
+  bool Has(const query::EdgeAccessor &edge,
+           query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
-  bool Accept(const memgraph::query::DbAccessor &dba, const std::vector<memgraph::storage::LabelId> &labels,
-              query::AuthQuery::FineGrainedPrivilege fine_grained_permission) const override;
+  bool Has(const std::vector<memgraph::storage::LabelId> &labels,
+           query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
-  bool Accept(const memgraph::query::DbAccessor &dba, const memgraph::storage::EdgeTypeId &edge_type,
-              query::AuthQuery::FineGrainedPrivilege fine_grained_permission) const override;
+  bool Has(const memgraph::storage::EdgeTypeId &edge_type,
+           query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
-  bool HasGlobalPermissionOnVertices(
+  bool HasGlobalPrivilegeOnVertices(
       memgraph::query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
-  bool HasGlobalPermissionOnEdges(
+  bool HasGlobalPrivilegeOnEdges(
       memgraph::query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) const override;
 
  private:
   auth::User user_;
+  const memgraph::query::DbAccessor &dba_;
```

Comment:
Replace `const &` with `const *` because it's more clear that you are holding a pointer